HeterogeneousCore/AlpakaUtilities/interface/VecArray.h

```diff
@@ -0,0 +1,107 @@
+#ifndef AlpakaCore_VecArray_h
+#define AlpakaCore_VecArray_h
+
+//
+// Author: Felice Pantaleo, CERN
+//
+
+#include <utility>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
```

Comment:
Can this be replaced as
```suggestion
#include <alpaka/alpaka.hpp>
```
?